src/features/Search/Search.tsx

```diff
@@ -331,11 +238,10 @@ class SearchContainer extends React.Component<{}, ISearchState> {
       '',
       window.location.href.split('?')[0] + newSearch
     );
-  }
+  };
 
-  private filter() {
-    const { sponsor, viewSaved, results } = this.state;
-    const searchBar = this.state.searchBar.toLowerCase();
+  const filter = () => {
+    const currSearchBar = searchBar.toLowerCase();
```

Comment:
what's the diff between `searchBar` and `currSearchBar`?

Reply:
I would assume that it's just `searchBar` but lowercase?

IIRC this function is just to filter out rows that don't match the search bar.